md/forms.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+
+from django import forms
+from django.db.models import Q
+from selectable.forms import AutoCompleteWidget, AutoCompleteSelectField
+
+from md import models as stops
+from md.lookups import AgencyLookup
+
+
+def addNoneOpt(choices):
+    opts = list(choices)
+    opts.insert(0, (None, "---"))
+    return opts
+
+
+class SearchForm(forms.Form):
+    agency = forms.CharField(
+        label='Agency Name',
+        widget=AutoCompleteWidget(AgencyLookup),
+        help_text="ex: Durham Police Department")
+    officer = forms.CharField(
+        required=False,
+        help_text="ex: 227")
+    # start_date = forms.DateField(
+    #     required=False,
+    #     help_text="ex: 8/13/2013")
```

Comment:
It isn't clear to me what search criteria should be enabled, and the enabled fields don't seem to match the code in get_query().  Is this search form usable at all as of this pull request?  If not, I think we shouldn't deliver it at all yet.